### ISSUE TITLE
MDS description rewording.

### ISSUE THREAD

**Issue by epebe** [CleverRaven/Cataclysm-DDA#76729]
### Is your feature request related to a problem? Please describe.

Recently, I proposed removing the MDS's torso strapped layer which was rejected in #76711. However, I found some inconsistencies between the description and the item that I would like to bring up here. 

Per @John-Candlebury, 
> The MDS covers the strapped layer because that's where both the poncho and the integrated bags are placed. 

Thing is, the strapped layer only covers the torso. The arms and legs are covered at the outer layer, so it is not clear what covers the arms and legs if the strapped layer represents the poncho. A poncho is generally a sheet of fabric with a head hole draped over the body, so I don't see how it could turn from strapped to outer on the arms. Also, if the torso strapped layer is the poncho, what covers the outer layer on the torso?

Frankly, I just don't think something like this can be considered the outer layer or have body-fitting rigid armor attached to it. It has no sleeves, is very loose, and is draped over the body; it's more like a cape or cloak than a jacket, which are all strapped in game. 
<img width="325" alt="Captura de pantalla 2024-09-29 a la(s) 9 08 14 p m" src="https://github.com/user-attachments/assets/100f156b-0234-4ab9-ab7e-716fd5450b78">

My interpretation is there is an underlayer covering the arms, legs, and torso where armor is attached (the outer layer in game). Something like a gambeson, since it also uses the outer layer, covers the arms, and is draped over the legs. The inserts would go on this underlayer, and the poncho should go above it on the torso strapped layer. 

To me, this makes sense considering the mantle and skirt are outer layer, which would go under the poncho's strapped layer.

### Solution you would like.

Reword the description and change thickness values. 

My proposal: 
1. Reword the description to emphasize that the inserts attach to something else underneath the poncho. 

- Original: "A custom-designed poncho for the true frontier explorer, made with aramid fibers and robust ripstop nylon.  Striped in brown and blue, the white seal of Hub 01 is emblazoned proudly on the front.  It features several quick access pouches attached to a waist drawstring, and two waterproof dry bags for carrying liquids.  It also features several tight clamps and velcro strips around the waist and shoulders; perhaps something attaches to them."

- Suggested: "**_Custom-designed armor_** for the true frontier explorer, made with aramid fibers and robust ripstop nylon.  Striped in brown and blue, the white seal of Hub 01 is emblazoned proudly on the front.  It features several quick access pouches attached to a waist drawstring and two waterproof dry bags for carrying liquids.  **_Several tight clamps and velcro strips around the waist and shoulders are covered by a large poncho._**"

2. Increase the torso armor thickness to reflect the poncho's additional coverage and/or reduce the limb armor thickness.
eg, 3 -> 5 mm on torso and 3 -> 2 mm on arms/legs to represent 2mm underlayer and 3mm poncho. 

3. Add the strapped layer to the waist to reflect the waist drawstring pouches. 

### Describe alternatives you have considered.

Alternatively, have the MDS cover the strapped layers of the arms and legs too if the poncho does cover everything. This still leaves the issue of the poncho covering the outer layer, and I don't see how one item of fabric can cover the outer and strapped layers. 

### Additional context

The issue with this proposal is that it offers no rationale to keep the poncho integrated. It is not necessary to attach armor, and though it provides extra protection, it conflicts with rigging. Some survivors may prefer to have additional storage options, and not letting them swap it out defeats the purpose of being modular. But I understand the poncho is an integral part of the original concept and featured in much fanart, so this is a tentative suggestion:

Make the poncho a separate piece wearable with/attachable to the MDS.

**Comment by IdleSol** [comment]
In my opinion you are misrepresenting this piece of armor. MDS is a short jumpsuit that takes up the outer layer. It has pockets and attachments for the mantle and skirt. And here they are already located on the Belted layer.

Plus it's a balancing system. The MDS pockets give a volume of 1.5 x 2 + 1.0 x 2 = 5 liters. If you add a harness to them, it's overkill.

I envision it something like this:
![1](https://github.com/user-attachments/assets/5e4d518a-1dfb-4ec4-a27b-eb5d7c33f0d6)

Theoretically, you could:
1. Divide the torso into two parts ( upper and lower). Remove the belted flag for the upper part, leaving only for the lower part (belted pouches). Also add belted for the hips. This would allow harness to be worn for the top. (My opinion, too good to do).
2. Make an alternate version of the MDS, with no pockets. Just pockets for ablative armor.

**Comment by epebe** [comment]
Thanks for your input. Yes, I could be misrepresenting the MDS with my interpretation, which is why it should be reworded for more clarity. 

As it stands, the MDS description only mentions a poncho, with no jumpsuit or underlayer like you and I mentioned. This creates issues of versimilitude because it's hard to imagine armor being attached to the inside of a loose cloak rather than to something strapped around the body. 

What I'm proposing is to make the description emphasize that there's a layer underneath the poncho where the armor and bags are attached. I understand the balance reasons for not being molle compatible, so I am not proposing to remove the torso strapped layer or add molle at the moment.

**Comment by Kamejeir** [comment]
I haven't read too closely anything here, but see the topic, and figure I should share my earlier findings: I got [this](https://old.reddit.com/r/cataclysmdda/comments/xwp0og/hub_mercenary_gear/) thrown my way. Made by candlebury.

**Comment by epebe** [comment]
Yeah, I saw that on the reddit a few years back and considered it while making this post. I don't see contradictions with what I wrote. 

In the picture, the poncho ends at the shoulders, and there's something else covering the upper arms and lower thighs. Either there's an underlayer to the MDS, or there is no coverage in those locations and it's the hubsuit visible. 

Two, there are no armor plates visible. Either the character isn't wearing any, or they are strapped underneath the poncho. As mentioned above, I don't think it's practical to secure rigid body fitting armor to something as loose as a poncho, so I propose emphasizing the underlayer as where armor is attached.

**Comment by fairyarmadillo** [comment]
I don't think a poncho makes any sense as the basis for a suit of armor. It is definitionally outerwear, and trying to stack things on top of one would lead to a lot of extra fabric restricting movement as it was pinched around the wearer's body.

IMO it would work better and be less confusing as a vest or jacket with ESAPI-style inserts. A protective skirt or mantle could easily fit over such an item without causing issues.

**Comment by epebe** [comment]
Thanks. I completely agree. The reason I tried to keep the poncho is that last time a rework was proposed, https://github.com/CleverRaven/Cataclysm-DDA/issues/66410, the realism arguement didn't win over the original design. 

Honestly, this issue has been up for a few days; I figure this discussion is all there's to be had. I'll go ahead with a PR if there's no objections by next week.

**Comment by IdleSol** [comment]
I'm in favor of your version of the description, but without the last line:

>"Custom-designed armor for the true frontier explorer, made with aramid fibers and robust ripstop nylon. Striped in brown and blue, the white seal of Hub 01 is emblazoned proudly on the front. It features several quick access pouches attached to a waist drawstring and two waterproof dry bags for carrying liquids. <s>Several tight clamps and velcro strips around the waist and shoulders are covered by a large poncho."</s>
